src/renderer/patches/platformClass.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vencord Desktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import { addPatch } from "./shared";
+
+addPatch({
+    patches: [
+        {
+            find: "platform-web",
+            replacement: {
+                // eslint-disable-next-line no-useless-escape
+                match: /return (?=__OVERLAY__\?""\.concat\((\i))/,
+                replace: "$1=$self.getPlatformClass(); return "
+            }
+        }
+    ],
+
+    getPlatformClass() {
+        const platform = navigator.platform.toLowerCase();
+
+        if (platform.includes("mac")) return "platform-osx";
+        if (platform.includes("win")) return "platform-win";
```

Comment:
does this not cause issues? Since now it expects there to be the entire Discord titlebar

Reply:
Yes, platform-win leads to there being no padding at the top 
![image](https://github.com/Vencord/Vesktop/assets/45497981/eab376b7-ccea-4702-879c-4e8683865fb8)

platform-linux on the other hand has no effect. As such I would remove the windows and linux logic and only use the osx class